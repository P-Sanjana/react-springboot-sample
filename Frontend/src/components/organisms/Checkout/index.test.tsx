import React from 'react';
import { render } from '@testing-library/react';
import CheckoutComponent from './index';
import { MemoryRouter } from 'react-router-dom';

describe('Checkout Component', () => {
  it('should match snapshot', () => {
    const wrapper = render(
      <MemoryRouter>
        <CheckoutComponent total='0.0234510 BTC' process='buy' />,
      </MemoryRouter>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
