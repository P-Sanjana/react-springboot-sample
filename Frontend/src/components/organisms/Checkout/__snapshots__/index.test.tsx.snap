// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkout Component should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiBox-root MuiBox-root-11 makeStyles-parentBox-10"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-justify-content-xs-center"
        >
          <div
            class="MuiGrid-root makeStyles-imgGrid-2 MuiGrid-item MuiGrid-grid-xs-12"
          >
            <img
              alt="Image"
              class="makeStyles-image-1"
              src="check-line.svg"
            />
          </div>
          <div
            class="MuiGrid-root makeStyles-total-3 MuiGrid-item MuiGrid-grid-xs-12"
          >
            <h4
              class="MuiTypography-root makeStyles-btc-4 MuiTypography-h4"
            >
              0.0234510 BTC
               
            </h4>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
          >
            <div
              class="MuiBox-root MuiBox-root-12 makeStyles-buysellcontent-5"
            >
              <p
                class="MuiTypography-root makeStyles-content-6 MuiTypography-body2"
              >
                Purchase is completed, please check your balance in your crypto wallet
              </p>
            </div>
          </div>
          <div
            class="MuiBox-root MuiBox-root-13 makeStyles-buttons-7"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-button1-8 MuiButton-outlinedPrimary"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                <span
                  class="MuiTypography-root MuiTypography-caption"
                >
                  Buy Crypto
                </span>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <a
              href="/wallet"
              style="text-decoration: none;"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button2-9 MuiButton-containedPrimary"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiButton-label"
                >
                  <span
                    class="MuiTypography-root MuiTypography-caption"
                  >
                    Go To Wallet
                  </span>
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
            </a>
          </div>
        </div>
      </div>
      ,
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiBox-root MuiBox-root-11 makeStyles-parentBox-10"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-justify-content-xs-center"
      >
        <div
          class="MuiGrid-root makeStyles-imgGrid-2 MuiGrid-item MuiGrid-grid-xs-12"
        >
          <img
            alt="Image"
            class="makeStyles-image-1"
            src="check-line.svg"
          />
        </div>
        <div
          class="MuiGrid-root makeStyles-total-3 MuiGrid-item MuiGrid-grid-xs-12"
        >
          <h4
            class="MuiTypography-root makeStyles-btc-4 MuiTypography-h4"
          >
            0.0234510 BTC
             
          </h4>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
          <div
            class="MuiBox-root MuiBox-root-12 makeStyles-buysellcontent-5"
          >
            <p
              class="MuiTypography-root makeStyles-content-6 MuiTypography-body2"
            >
              Purchase is completed, please check your balance in your crypto wallet
            </p>
          </div>
        </div>
        <div
          class="MuiBox-root MuiBox-root-13 makeStyles-buttons-7"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-button1-8 MuiButton-outlinedPrimary"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiButton-label"
            >
              <span
                class="MuiTypography-root MuiTypography-caption"
              >
                Buy Crypto
              </span>
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
          <a
            href="/wallet"
            style="text-decoration: none;"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button2-9 MuiButton-containedPrimary"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                <span
                  class="MuiTypography-root MuiTypography-caption"
                >
                  Go To Wallet
                </span>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </a>
        </div>
      </div>
    </div>
    ,
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
